<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarEntrada</class>
 <widget class="QDialog" name="RegistrarEntrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Registrar entrada de cliente:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cliente: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCliente"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCliente"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditInformacion">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRegistrar">
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
